docs/_posts/2020-01-24-533-release.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: blogpost
+title:  "5.3.3 Release Announcement"
+section_id: blog
+date:   2020-01-24 00:00:00
```

Comment:
should we update the date to today ?

Reply:
We could, but the release tag was the 24th...